Added bubble components and wdio tests

sdadn opened the pull request:
### Summary
<!--- Summarize and explain the reason behind these code changes. What are the changes, and why are they necessary? -->
Created bubble components and wdio tests.

<!--- Include any issue addressed by this pull request. -->
<!--- Example: Closes #45 -->
Closes #95 

### Deployment Link
<!---Include the deployment link, if applicable. -->
<!--- Example: https://terra-graphs-deployed-pr-45.herokuapp.com/ -->
https://terra-graphs-deployed-pr-#.herokuapp.com/

### Testing
<!-- Demonstrate that these changes are stable. How have these changes been verified? -->

### Additional Details
<!-- List anything else that is relevant to this issue. Additional information will help us better understand your changes and speed up the review process. -->

<!--
*Before publishing*

1. Assign yourself to the PR.
2. Add the appropriate labels
3. Add your name to the CONTRIBUTORS.md file. Adding your name to the CONTRIBUTORS.md file signifies agreement to all rights and reservations provided by the License.
-->

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

yuderekyu commented on packages/terra-graphs/src/components/Bubble/BubbleSingleDataset.jsx:
```diff
@@ -0,0 +1,60 @@
+}) => {
+  React.useEffect(() => {
+    const graph = Carbon.api.graph(graphConfig);
+    if (!(utils.isUndefined(dataset))) {
+      if (!(utils.isUndefined(timeout))) {
```
```suggestion
    if (dataset) {
      if (timeout) {
```

yuderekyu commented on packages/terra-graphs-docs/src/example-datasets/popup.js:
```diff
@@ -0,0 +1,291 @@
+    item.append("label").classed("popup-label", true).text(label);
+    item.append("g").classed("popup-text", true).text(value);
+};
+const createTrackItem = (pair, label, items) => {
```
I made a similar comment on a separate graph pull request, but I think converting most of these functions into an object based api will really help clarify what each function argument represents.
```suggestion
const createTrackItem = ({ pair, label, items }) => {

// useage
createTrackItem({ pair: element, label: 'real-world-label', items: dataArr });
```

sdadn replied:
I agree, but I think that's beyond the scope of this issue. The library has some questionable design choices and refactoring those is a separate project that will take more than an iteration. I'd rather avoid making api design changes until we're ready to tackle it as dedicated project due to the spaghetti nature of the implementation and avoid breaking the library.

yuderekyu replied:
Here are some key questions - is this pull request moving old code into the terra-graphs repo, or are these functions brand new on your side? If it's the former then I agree with you, but if it's the latter, I disagree and think it's in our best interest to update the api. 

Writing some context in the pr description would help everyone understand the intent of the pull request. Like "migrating bubble examples from carbon into terra-graphs to render on dev site for integration tests"

sdadn replied:
I apologize for the lack of clarity. These functions and datasets are being migrated from the old dev-site.

sdadn commented:
@tbiethman @yuderekyu @AshishMotanamGurunadham  I believe I finished updated comments. A few updates to the PR:
1. There was a typo in one of the graph folders (piard results). Moved them to the correct folder.
2. Had to delete and rerun all wdio tests, so you may see some updates for other graphs. There are no changes but git detects them as new files.

AshishMotanamGurunadham commented:
> @tbiethman @yuderekyu @AshishMotanamGurunadham I believe I finished updated comments. A few updates to the PR:
> 
> 1. There was a typo in one of the graph folders (piard results). Moved them to the correct folder.
> 2. Had to delete and rerun all wdio tests, so you may see some updates for other graphs. There are no changes but git detects them as new files.

May I know the reason why you had to generate new images for all tests? I mean will we get conflicts when merged with main?

And can you also move dev.module.scss file to terra-graphs-docs(Just merge with main). https://github.com/cerner/terra-graphs/tree/main/packages/terra-graphs-docs/src/Css

yuderekyu commented on packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/BubbleGraph/ColorBasedGraph.jsx:
```diff
@@ -0,0 +1,20 @@
+const dataset = utils.deepClone(bubbleDataColorBased);
+
+export default () => (
+  <React.Fragment>
```
Not a big deal, but I just want to point out that React has a shorthand syntax for fragments
```suggestion
  <>
  </>
```

sdadn commented:
@AshishMotanamGurunadham I have rebased and updated the scss file imports here: 4b40fb5cb0cabecb4192d706b5b807166ea9ed01
There shouldn't be any merge conflicts with main as Github detects the old files as being deleted and new ones being added.
